### DISCUSSION TITLE
Assistance Needed: Regaining Access to GitHub Account<title>

### DISCUSSION THREAD

**Discussion by shrutikhare18** [community/community#116682] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this email finds you well. I am writing to seek assistance with regaining access to my GitHub account.

Unfortunately, the email address associated with my GitHub account is no longer accessible as it was linked to my former office email [redacted], username - **shruti-zs**,  which I no longer have access to due to my recent resignation from ZopSmart. As a software developer, access to my GitHub account is crucial for managing repositories and projects essential to my work.

I have attempted to update the email associated with my GitHub account but have encountered difficulties due to the verification processes involved. I kindly request your assistance in helping me regain access to my account or providing guidance on the necessary steps to resolve this issue.

Please let me know if there are any specific documents or information required from my end to verify my identity and facilitate the process of updating the email associated with my GitHub account.

Your prompt attention to this matter would be greatly appreciated as it impacts my ability to manage my work effectively. Thank you in advance for your assistance.

Looking forward to your prompt response.

Warm regards,

Shruti Khare
shruti-zs
[redacted]

**Comment by queenofcorgis** [comment, marked as answer]
Hi @shrutikhare18 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

Please note: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
